data.tf

```diff
@@ -89,8 +89,7 @@ locals {
   # - get cidr block value from AWS IPAM
   # - create flow logs
 
-  # # if var.vpc_id is passed, assume create = `false` and cidr comes from data.aws_vpc
-  create_vpc = var.vpc_id == null ? true : false
+  create_vpc = var.create_vpc ? true : false
```

Comment:
You shouldn’t need a bool local that’s just a ternary bool 

just replace local.create_vpc with the variable

Reply:
That's true, I've replaced it.

Reply:
I've been using this branch to successfully build around 10 vpc's so far. Code changes are good. @drewmullen are we ready to merge?